### PR TITLE
Feat/todo

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Todos for the user
- Added Search, Filter, Sort, and Clear functionalities for todos.
- Editing todos
- Deleting todos

## Screenshot 
image
image

### CODE REVIEW

File: src/pages/Todo/Todo.jsx [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+                                  })
+                                }
+                              ></i>
+                              <i
+                                className="far fa-trash text-4xl visible text-black cursor-pointer"
+                                onClick={() =>
+                                  todoDispatch({
+                                    type: "DELETE_TODO",
+                                    payload: item,
+                                  })
```
Comment (rohanmathur91) [inline src/pages/Todo/Todo.jsx:303]: suggestion: have a handler which does this and provide that to onClick

**Review by rohanmathur91 — APPROVED** [review]
- Clean code and good naming convention.
- Added some suggestions, if you want you can ignore them for now.

**Review by anupkgurung — APPROVED** [review]
nice use of utilities.
suggestion : you can have onclick handlers to do the onclick part.
rest seems good.

**Comment by Aditi0505** [comment]
> * Clean code and good naming convention.
> * Added some suggestions, if you want you can ignore them for now.

Thank you for your suggestions! I will add all the suggestions in next iteration, when I have some bandwidth.

**Comment by Aditi0505** [comment]
> nice use of utilities. suggestion : you can have onclick handlers to do the onclick part. rest seems good.

Thank you for your suggestions! I will add all the suggestions in next iteration, when I have some bandwidth.

**Pull request merged**